index.html

```diff
@@ -4,67 +4,80 @@
+    />
     <link rel="stylesheet" href="style.css" />
     <title>Kehinde Onifade Portfolio</title>
   </head>
   <body>
     <div class="container">
-      <header class="header">
-        <a href="#" class="welcome">Welcome</a>
-        <nav class="hamburger">
-          <img src="assets/images/burger.png" alt="nav" />
-        </nav>
-      </header>
+      <div class="contain">
+        <div class="background-stripe"><img src="assets/images/all.svg" alt="design" /></div>
+        <header class="header">
+          <a href="#" class="welcome">Welcome</a>
 
-      <div class="design-container">
-        <img class="des1" src="assets/images/des1.png" alt="" />
-        <img class="des2" src="assets/images/des2.png" alt="" />
-        <img class="des3" src="assets/images/des3.png" alt="" />
-        <img class="des4" src="assets/images/des4.png" alt="" />
-        <img class="des5" src="assets/images/des5.png" alt="" />
-      </div>
-
-      <section class="headline">
-        <h1 class="introduction">
-          Hey there. I’m Kehinde <br /><span> I’m a software developer</span>
-        </h1>
-
-        <p class="support-text">
-          I can help you build a product , feature or website Look through some of my work and
-          experience! If you like what you see and have a project you need coded, don’t hestiate to
-          contact me.
-        </p>
-        <div class="social-links">
           <ul>
-            <li>
-              <a href="https://twitter.com/RideOnOne09"
-                ><img src="assets/images/twitter.svg" alt="twitter"
-              /></a>
-            </li>
-            <li>
-              <a href="https://www.linkedin.com/in/kehindeonifade/"
-                ><img src="assets/images/linkedin.svg" alt="linkedIn"
-              /></a>
-            </li>
-            <li>
-              <a href="https://medium.com/@onifkay"
-                ><img src="assets/images/medium.svg" alt="medium"
-              /></a>
-            </li>
-            <li>
-              <a href="https://github.com/kendoriddy"
-                ><img src="assets/images/github.svg" alt="github"
-              /></a>
-            </li>
-            <li>
-              <a href="https://angel.co"
-                ><img src="assets/images/angellist.svg" alt="angellist"
-              /></a>
-            </li>
+            <li>Portfolio</li>
+            <li>About</li>
+            <li>Contact</li>
           </ul>
+
+          <nav class="hamburger">
+            <img src="assets/images/burger.png" alt="nav" />
+          </nav>
+        </header>
+
+        <div class="design-container">
+          <img class="des1" src="assets/images/des1.png" alt="" />
+          <img class="des2" src="assets/images/des2.png" alt="" />
+          <img class="des3" src="assets/images/des3.png" alt="" />
+          <img class="des4" src="assets/images/des4.png" alt="" />
+          <img class="des5" src="assets/images/des5.png" alt="" />
         </div>
```

Comment:
- [ _Optional_] Kindly ensure that you use `background-image` to implement decorations in the background. Decorations are always supposed to be background- images since using images for designs is non-intuitive and will make screen readers think it's an image showing something important while it's actually just a decoration.

Reply:
What can I do in this case please. @zhadier

Reply:
Hey @kendoriddy, Yes it does look perfect on a specific viewport, but please do keep in mind that desktops come in a variety of sizes so your design needs to be responsive and not break when the screen size is a little bigger or smaller. Right now you're using position absolute to fix the position of the background for  your screen. Instead you should add it as a background image for the section with `background-position: center`

Reply:
That way it will always be centered in the section no matter what screen size